src/com/maddyhome/idea/vim/helper/MessageHelper.kt

```diff
@@ -23,9 +23,11 @@ import org.jetbrains.annotations.NonNls
 import org.jetbrains.annotations.PropertyKey
 
 @NonNls
-private const val BUNDLE = "messages.IdeaVimBundle"
+private const val IDEAVIM_BUNDLE = "messages.IdeaVimBundle"
 
-object MessageHelper : AbstractBundle(BUNDLE) {
+object MessageHelper : AbstractBundle(IDEAVIM_BUNDLE) {
+
+  const val BUNDLE = IDEAVIM_BUNDLE
```

Comment:
Sorry, why this variable was introduced?

Reply:
Maybe it was to get the IDE inspections working to show the actual value of IDs passed to `MessageHelper.message`? But I don't really remember, and can't see how that change would help. Sorry...